extensions/ql-vscode/src/view/results/result-tables.tsx

```diff
@@ -168,10 +164,7 @@ export class ResultTables extends React.Component<
             getResultSets(props.rawResultSets, props.interpretation),
           );
 
-        return {
```

Comment:
This is the only bit where it isn't perfectly replicated on the new component and I'm unsure if it's needed or not.

This is checking if the result set exists and if not setting the selected table and page to default values. I'm not sure how to trigger this state. I also think it should be covered by the `useEffect` hook in the header component which always sets the selected page to the value from the parsed result set.